commands/label_errors.py

```diff
@@ -106,14 +220,13 @@ def main(
     """
     kili = Kili(api_key=api_key, api_endpoint=api_endpoint)
     input_type, jobs, _ = get_project(kili, project_id)
+    jobs = curated_job(jobs, target_job, ignore_job)
 
     for job_name, job in jobs.items():
-        if target_job and job_name not in target_job:
-            continue
         kili_print(f"Detecting errors for job: {job_name}")
         content_input = get_content_input_from_job(job)
-        ml_task: MLTaskT = job.get("mlTask")  # type: ignore
-        tools: List[ToolT] = job.get("tools")
+        ml_task = job.get("mlTask")
```

Comment:
is there a reason why the types are removed?

Reply:
No longer needed

Reply:
job is now better typed